features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher-new/source/src/app/components/Apis/Details/Subscriptions/SubscriptionPoliciesManage.jsx

```diff
@@ -0,0 +1,173 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { Component } from 'react';
+import { FormattedMessage, injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import withStyles from '@material-ui/core/styles/withStyles';
+import Checkbox from '@material-ui/core/Checkbox';
+import FormLabel from '@material-ui/core/FormLabel';
+import FormControl from '@material-ui/core/FormControl';
+import FormGroup from '@material-ui/core/FormGroup';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Alert from 'AppComponents/Shared/Alert';
+import Progress from 'AppComponents/Shared/Progress';
+import API from 'AppData/api';
+
+const styles = theme => ({
+    subscriptionPoliciesPaper: {
+        marginTop: theme.spacing.unit * 2,
+        paddingLeft: theme.spacing.unit * 2,
+        paddingBottom: theme.spacing.unit * 2,
+    },
+    grid: {
+        margin: theme.spacing.unit * 1.25,
+    },
+    gridLabel: {
+        marginTop: theme.spacing.unit * 1.5,
+    },
+});
+
+/**
+ * Manage subscription policies of the API
+ * */
+class SubscriptionPoliciesManage extends Component {
+    constructor(props) {
+        super(props);
+        this.api = props.api;
+        this.state = {
+            subscriptionPolicies: {},
+            selectedSubscriptionPolicies: props.api.policies,
+            updateInProgress: false,
+        };
+        this.handleChange = this.handleChange.bind(this);
+    }
+
+    componentDidMount() {
+        API.policies('subscription')
+            .then((res) => {
+                this.setState({ subscriptionPolicies: res.obj.list });
```

Comment:
Use the response `body` instead of `obj`, `obj` is a [legacy API](https://github.com/swagger-api/swagger-js) will remove in future releases.

Reply:
Fixed